cmd/influx_stress/influx_stress.go

```diff
@@ -29,32 +31,36 @@ func main() {
 		defer pprof.StopCPUProfile()
 	}
 
-	c, err := stress.NewConfig(*config)
-	if err != nil {
-		fmt.Println(err)
-		return
-	}
+	if *useV2 {
+		v2.RunStress(*config)
+	} else {
 
-	if *db != "" {
-		c.Provision.Basic.Database = *db
-		c.Write.InfluxClients.Basic.Database = *db
-		c.Read.QueryClients.Basic.Database = *db
-	}
+		c, err := stress.NewConfig(*config)
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
 
-	w := stress.NewWriter(&c.Write.PointGenerators.Basic, &c.Write.InfluxClients.Basic)
-	r := stress.NewQuerier(&c.Read.QueryGenerators.Basic, &c.Read.QueryClients.Basic)
-	s := stress.NewStressTest(&c.Provision.Basic, w, r)
+		if *db != "" {
```

Comment:
Yeah. Its because of the indentation change to add the new stress test that it doesn't line up with the rest. If you look at the whole file and not just the `diff` it should read properly.